src/NUnitFramework/framework/Constraints/CollectionEquivalentConstraint.cs

```diff
@@ -112,5 +112,30 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
             base.Using(EqualityAdapter.For(comparison));
             return this;
         }
+
+
+        /// <summary>
+        /// Flag the constraint to use the supplied Comparison object.
+        /// </summary>
+        /// <param name="comparison">The Comparison object to use.</param>
+        #pragma warning disable CS0612
+        [Obsolete]
+        public new CollectionItemsEqualConstraint Using<T>(Comparison<T> comparison)
+        {
+            Comparer.ExternalComparers.Add(EqualityAdapter.For(comparison));
+            return this;
+        }
+
```

Comment:
I would have expected a `#pragma warning restore CS0612` here

Reply:
The warnings break the build.

Reply:
Oh, restore, right :)